lib/wallet/src/Cardano/Wallet/Write/Tx/Balance.hs

```diff
@@ -530,15 +511,16 @@ balanceTransactionWithSelectionStrategyAndNoZeroAdaAdjustment
     -> ExceptT ErrBalanceTx m (Cardano.Tx era, changeState)
 balanceTransactionWithSelectionStrategyAndNoZeroAdaAdjustment
     tr
-    (UTxOAssumptions txLayer toInpScriptsM mScriptTemplate txWitnessTag)
+    txWitnessTag
+    utxoAssumptions
     protocolParameters@(ProtocolParameters pp)
     timeTranslation
     (UTxOIndex walletUTxO internalUtxoAvailable cardanoUTxO)
     genChange
     s
     selectionStrategy
-    ptx@(PartialTx partialTx inputUTxO redeemers)
-    = do
+    ptx@(PartialTx partialTx inputUTxO redeemers) = do
+    --
```

Comment:
Perhaps a stray comment?
```suggestion
```

Reply:
We had a discussion about this controversial embellishment with @Anviking: the idea is that I want to visually separate a list of arguments from the function body.  I am still in favor of leaving and using it in similar situations, but if you dislike it I won't insist.